src/faction_camp.cpp

```diff
@@ -343,6 +343,34 @@ void talk_function::camp_missions( mission_data &mission_key, npc &p )
         }
     }
 
+    if( bcp->has_level( camp_ctr, 5, base_dir ) ) {
+        std::vector<std::shared_ptr<npc>> npc_list = companion_list( p, "_faction_camp_clearcut" );
+        mission_key.text["Clearcut"] = string_format( _( "Notes:\n"
+                                       "Send a companion to a clear a nearby forest.\n \n"
+                                       "Skill used: fabrication\n"
+                                       "Difficulty: 1 \n"
+                                       "Effects:\n"
+                                       "> 95%% of trees/trunks at the forest position"
+                                       " will be cut down.\n"
+                                       "> 0%% of total material will be brought back.\n"
+                                       "> Forest will become a field tile.\n"
+                                       "> Useful for clearing land for another faction camp.\n \n"
```

Comment:
You might also mention here whether or not it will clear enough space to make a road. Will enough trees get cut down that I can drive a normal sized car through?